manager/allocator/network.go

```diff
@@ -857,13 +857,25 @@ func (a *Allocator) doTaskAlloc(ctx context.Context, ev events.Event) {
 	log.G(ctx).Debugf("task %v was marked pending allocation", t.ID)
 }
 
+func nodeIsWindows(node *api.Node) bool {
+	return node != nil &&
+		node.Description != nil &&
+		node.Description.Platform != nil &&
+		node.Description.Platform.OS == "windows"
+
+}
+
```

Comment:
unrelated: for extra credit, also add a comment for this function ? :)

Reply:
I'll give it a shot, but I'm only half-certain that I understand its purpose and role in the whole system myself.  :)